### PR TITLE
hello world from C++

### PR DESCRIPTION
**Pull request by VishalBheda**

### CODE REVIEW

**Comment by captain-pool** [comment]
Hey @VishalBheda thanks for the contribution! Can you put the files in a folder with the language name and then push it?

File: c++/hello-world.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	cout<<"Welcome "<<user;
+
+    for(int i=0; i<5; i++){
+        for(int j=0; j<i; j++){
+            cout<<"*";
+        }
+        cout<<"\n";
+    }
+    return 0;
+}
```
Comment (syed-awais-ali) [inline c++/hello-world.cpp]: missing new line at the EOF

File: hello-world.cpp [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#include<iostream>
+
+using namespace std;
+
+int main(){
+    cout<< "Hello World! \n";
+    return 0;
+}
```
Comment (syed-awais-ali) [inline hello-world.cpp]: missing new line at the EOF

**Comment by captain-pool** [comment]
@VishalBheda can you create 3 seperate problems in 3 different .cpp files?
and place them under
C++ folder?

**Comment by captain-pool** [comment]
Looks great! Thanks for the contribution! :)

**Pull request merged**